server/src/http/controllers/unsubscribeLbaCompany.controller.ts

```diff
@@ -23,43 +29,61 @@ export default function (server: Server) {
       },
     },
     async (req, res) => {
-      let result: "OK" | UNSUBSCRIBE_EMAIL_ERRORS = "OK"
+      let result: { result: "OK" | UNSUBSCRIBE_EMAIL_ERRORS; companies?: IUnsubscribeCompanyData[] } = { result: "OK" }
```

Comment:
Type en `IUnsubscribeQueryResponse` (à ajouter au model) ou bien :
```suggestion
      let result: z.output<typeof ZUnsubscribeQueryResponse> = { result: "OK" }
```